lib/core/styles/fonts/app_fonts.dart

```diff
@@ -1,3 +1,4 @@
-abstract class AppFonts{
-
+class AppFonts{
+  AppFonts._();
+  static const String kPrimaryFont = "Inter";
```

Comment:
not important to make it in a class

Reply:
with more fonts in one app, it isn't easy to use them